### PR TITLE
Draft roadmap

### PR DESCRIPTION
**Pull request by abarciauskas-bgse**

### CODE REVIEW

File: docs/roadmap.rst [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+Pangeo is looking to hire a part- or full-time communications manager who help with those forums as well as **creating a monthly newsletter**.
+
+This communications manager will also help define a strategy for attending and presenting at conferences attendance and organizing Pangeo-dedicated events. This communications manager will help define what, where and how we communicate about Pangeo. This communications manager will also help revamp the website.
+
+Supporting the Evoloution of the Geoscience Software Stack
+----------------------------------------------------------
+
+The Pangeo Steering Committee will drive the evolution of the Geoscience Software Stack. This will include:
+
+1. Setting higher level goals to guide the community in development and in writing proposals. For example, Pangeo advocates for the use of STAC to facilitate access to data. We believe setting higher level goals is a core function as a network of resources continues to evolve and sharing conventions is key to cross-platform collaboration.
```
Comment (paigem) [inline docs/roadmap.rst:34]:
```suggestion
1. _Advocate for new directions and best practices in the development of new tools, maintenance of existing tools, and the use of these tools in a geoscience context_: Advocacy will be driven by community input via community forums (e.g. Pangeo meetings or Discourse). We believe that establishing and communicating community-accepted conventions and best-practices is necessary for cross-platform collaboration as the landscape of resources continuously evolves. A current example is the Pangeo advocacy for the use of STAC to facilitate access to datasets.
```

**Review by paigem — CHANGES_REQUESTED** [review]
Thank you @abarciauskas-bgse for creating this document and laying the groundwork for the Pangeo roadmap. I made several suggested updates to expand on your points and emphasize the unique position that Pangeo is in and how we can leverage that to move forward.

Happy to discuss any of these points @abarciauskas-bgse or anyone else who wants to comment!

**Comment by jmunroe** [comment]
This might be too tactical, but is applying for funding like [NSF POSE](https://new.nsf.gov/funding/opportunities/pathways-enable-open-source-ecosystems-pose) (due September 7, 2023) be on the roadmap?

**Comment by paigem** [comment]
Thanks for your comment @jmunroe, and you bring up a good point! Should our Roadmap include a mention of writing funding proposals? That's certainly something we will need to do as a community. Since funding spans both (1) supporting the community and (2) supporting software, perhaps we can add a sentence about funding at the top of the roadmap?

Our first thought was to keep this roadmap pretty general, so my inclination is to not mention a specific funding call, but I'm open to adding it here if you think it makes sense @jmunroe.

**Comment by jmunroe** [comment]
A general roadmap is probably wiser than listing any specific funding calls. 

I agree that hiring a person makes sense as a goal. I'd advocate for a 'Community Manager' rather than (only) a 'Communications Manager' and include in the roadmap seeking funding to support that person full time.

**Comment by paigem** [comment]
@jmunroe I completely agree with the term "community manager", and indicated as such in my review of this PR. 😉

I can add a sentence about applying for funding to support the community manager.

**Review by paigem — APPROVED** [review]

**Pull request merged**